src/OpenTelemetry.Exporter.Geneva/GenevaLogExporter.cs

```diff
@@ -443,5 +489,66 @@ private static byte GetSeverityNumber(LogLevel logLevel)
                 return 1;
         }
     }
+
+    private static int Sanitize(byte[] buffer, int cursor, ref int cursorStartIdx, ref int validNameLength, string categoryName)
```

Comment:
Use `MethodImplOptions.AggressiveInlining`?

Reply:
Use a more descriptive method name.

Reply:
I'll have to do benchmarks to see whether `MethodImplOptions.AggressiveInlining` will have benefits here.
Would SanitizeCategoryName be a better name here?